test/unit/ipc/writer/stream-writer-tests.ts

```diff
@@ -37,6 +38,41 @@ import {
     Uint32,
     Vector
 } from 'apache-arrow';
+import { Codec, compressionRegistry } from 'apache-arrow/ipc/compression/registry';
```

Comment:
Are these not exported in the top-level export? It seems like they should be, since they'd allow users to register their own implementations?